server/src/daemon/prompter.rs

```diff
@@ -0,0 +1,210 @@
+// org.gnome.keyring.Prompter
+// https://gitlab.gnome.org/GNOME/gcr/-/blob/master/gcr/org.gnome.keyring.Prompter.xml
+
+use std::{
+    collections::HashMap,
+    sync::{Arc, Mutex},
+};
+
+use tokio;
+use zbus::{
+    fdo, interface,
+    message::Header,
+    proxy,
+    zvariant::{
+        self, DeserializeDict, ObjectPath, OwnedObjectPath, OwnedValue, SerializeDict, Type,
+    },
+    Connection, SignalContext,
+};
+
+use super::{secret_exchange, service_manager::ServiceManager};
+
+// May be change this to /org/oo7_daemon/Prompt
+const SECRET_PROMPTER_PREFIX: &str = "/org/gnome/keyring/Prompt/";
+
+#[derive(Clone, Default, DeserializeDict, Debug, Type, SerializeDict)]
+#[zvariant(signature = "dict")]
+pub struct Properties {
+    title: Option<String>,
```

Comment:
Where those properties are defined? what uses them at the end? are all of them still used?

Reply:
This is now removed :)